conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -1392,7 +1405,11 @@ TEST_F(
 TEST_F(
     zetMetricStreamerTest,
     GivenValidMetricGroupWhenTimerBasedStreamerIsCreatedThenExpectStreamerToGenrateCorrectNumberOfReports) {
-
+  
+  /* The time in seconds for the buffer to overflow would be 2 * (notifyEveryNReports * (samplingPeriod/nanoSecToSeconds))
+   * For this test it will be 9 seconds. The execution time between metric_streamer_open_for_device and synchronize observed on average is less than 50% of this
+   * which ensure this limit is never likely to hit even on the slowest of platform
```

Comment:
"   * which ensure this limit is never likely to hit even on the slowest of platform" coudl be avoided, since we donot know the slowest platform